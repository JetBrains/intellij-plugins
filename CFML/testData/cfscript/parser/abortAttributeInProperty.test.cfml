component accessors='true' {
    property name ="id" abort="true";
}
