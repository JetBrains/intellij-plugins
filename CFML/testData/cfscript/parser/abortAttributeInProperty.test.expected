CfmlFile:abortAttributeInProperty.cfml
  COMPONENT_DEFINITION
    PsiElement(COMPONENT_KEYWORD)('component')
    Attribute
      PsiElement(IDENTIFIER)('accessors')
      PsiElement(ASSIGN)('=')
      PsiElement(SINGLE_QUOTE)(''')
      AttributeValue
        PsiElement(STRING_TEXT)('true')
      PsiElement(SINGLE_QUOTE_CLOSER)(''')
    PsiElement(L_CURLYBRACKET)('{')
    PROPERTY
      PsiElement(IDENTIFIER)('property')
      NamedAttribute
        PsiElement(IDENTIFIER)('name')
        PsiElement(ASSIGN)('=')
        PsiElement(DOUBLE_QUOTE)('"')
        AttributeValue
          PsiElement(STRING_TEXT)('id')
        PsiElement(DOUBLE_QUOTE_CLOSER)('"')
      Attribute
        PsiElement(ABORT_KEYWORD)('abort')
        PsiElement(ASSIGN)('=')
        PsiElement(DOUBLE_QUOTE)('"')
        AttributeValue
          PsiElement(STRING_TEXT)('true')
        PsiElement(DOUBLE_QUOTE_CLOSER)('"')
    PsiElement(SEMICOLON)(';')
    PsiElement(R_CURLYBRACKET)('}')
